packages/schematics/angular/application/files/src/polyfills.ts.template

```diff
@@ -18,7 +18,7 @@
  * BROWSER POLYFILLS
  */
 
-/** IE11 requires the following for NgClass support on SVG elements */
+// IE11 requires the following for NgClass support on SVG elements
```

Comment:
The block comment is intention to aid in indicating which lines should be uncommented and which are informational.
If you would still like to adjust this so that the custom tslint rule set works, can you change this into a multi-line block comment instead?

Reply:
The thing is this is a custom tslint rule set from **Microsoft**, not mine. It could be fixed easily. This is the only reason I assumed that this should be fixed by default.

Reply:
The package `tslint-microsoft-contrib` is currently archived and no longer supported (more details: https://github.com/microsoft/tslint-microsoft-contrib/issues/876).

However, a changed similar to the following should allow the rule to pass without changing the comment type:
```
/** 
 * IE11 requires the following for NgClass support on SVG elements
 */
```

Reply:
Sure. So if you think it make sense to change it like that, I will do it. If you think this should be done individually (localy), feel free to close this MR.